src/main/java/com/pirimid/cryptotrade/websocket/coinbase/res/WSCoinbaseTradeDto.java

```diff
@@ -0,0 +1,28 @@
+package com.pirimid.cryptotrade.websocket.coinbase.res;
+
+import lombok.Data;
+import lombok.ToString;
+
+import java.util.Date;
+
+@Data
+@ToString
+public class WSCoinbaseTradeDto {
+    private String type;
+    private String trade_id;
+    private String taker_order_id;
```

Comment:
are we using camel case for naming or _?

Reply:
It is for exchange specific Dto that we will get from exchange from response so that's why I have kept the same naming as Exchange response.

Reply:
Can we follow the same convention as we have followed in other class by using @JsonProperty?